flink-runtime/src/test/java/org/apache/flink/runtime/net/ConnectionUtilsTest.java

```diff
@@ -61,7 +61,7 @@ public void testReturnLocalHostAddressUsingHeuristics() throws Exception {
 			assertNotNull(add);
 
 			// make sure that we returned the InetAddress.getLocalHost as a heuristic
-			assertEquals(InetAddress.getLocalHost(), add);
+			assertEquals(InetAddress.getByName("localhost"), add);
```

Comment:
How about also changing the blocker above to accept connections on any interface just to rule accidental successful resolve attempts out, too, i.e. using `ServerSocket blocker = new ServerSocket(0, 1, null)`?

Reply:
Thanks @NicoK review. Yes. I think this is a better way to use ```null``` instead of ```InetAddress.getLocalHost()``` to prevent multiple adapters cache expires from getting a  another result in the future. I will give a quick fix for that.

Reply:
Or, we can also still use ```InetAddress.getByName("localhost")```. Both ```null``` and ```InetAddress.getByName("localhost")``` are OKay.